lib/imagefs.coffee

```diff
@@ -280,3 +327,13 @@ exports.listDirectory = (definition) ->
 			listDirectory(disk, definition.partition, definition.path)
 	else if definition.image instanceof filedisk.Disk
 		listDirectory(definition.image, definition.partition, definition.path)
+
+###*
+# @summary Closes the allocated resources. Call this when you're done using imagefs if you expect the program to end.
+# @function
+# @public
+#
+# @returns {Promise}
+###
+exports.close = ->
+	ext2fs.close()
```

Comment:
> Nothing in resin-image-fs is calling ext2fs.close by itself.

Oh god, yeah :facepalm: – sorry, I totally mixed that up – don't mind this then.